team-config.yml

```diff
@@ -20,4 +20,14 @@ cft-platform:
     build_notices_channel: "#platops-build-notices"
   tags:
     application: "core"
-    businessArea: "CFT"    
\ No newline at end of file
+    businessArea: "CFT"
+
+soc:
+  team: "soc"
+  namespace: ""
+  slack:
+    contact_channel: "#dtspo-secops-chat"
```

Comment:
contact channel must be public, anyone should be able to contact the team from it, the dtspo one probably isn't the best one for this

Reply:
Asked Leon and still waiting for a response.  Likely they have none.  I've looked up all security related channels and devops-secops-chat seems most relevant.